main.c

```diff
@@ -134,6 +170,20 @@ main(int argc, char *argv[])
 	if (osUpdateScreenInit())
 		return -1;
 
+	/* Allow SIGTERM and SIGINT to interrupt pselect() and move to cleanup */
+	sigemptyset(&mask);
+	sigaddset(&mask, SIGTERM);
+	sigaddset(&mask, SIGINT);
+	sigfd = signalfd(-1, &mask, 0);
+	if (sigfd == -1) {
+		printf("Could not create signal fd\n");
```

Comment:
Already existing code is using printf() too. Just started to wonder: is stdout collected to journal or redirected to /dev/null, i.e. if errors happen are they discoverable from logs?

Reply:
Yes they are. Both stdout and stderr are collected to journal.